ember-app/app/mixins/events/issue.js

```diff
@@ -0,0 +1,31 @@
+import Ember from 'ember';
+
+var IssueEventMixin = Ember.Mixin.create({
+  hbevents: {
+    "{repositoryName} local.{model.number}.issue_closed": "closed",
+    "{repositoryName} local.{model.number}.issue_reopened": "opened"
+  },
+  _eventHandlers: {
+    closed: function(){
+     console.log("MESSAGE RECEIVED YO");
+     //this.get("model").set("state", message.issue.state);
+    },
+    opened: function(){
+     console.log("MESSAGE RECEIVED YO");
+     //this.get("model").set("state", message.issue.state);
+    },
+  },
```

Comment:
Re: ```_buildMeta```, I am not 100% sold on this pattern, essentially the messaging mixin uses it to map the local publish data, hard coding things like "type" might be too tight of a constraint